saboteur/server.go

```diff
@@ -20,7 +20,7 @@ func NewServer(db *sabotageDB) *Server {
 }
 
 // Listen starts a HTTP server on the provided host and port
-func (s *Server) Listen(host string, port string) {
+func (s *Server) Listen(host string, port string) []cap.Node {
```

Comment:
We probably want to receive the `http.Server` record here rather than create it ourselves. This way a client can decide to add security configurations to it.

Reply:
Uhm, but not sure if we can add an extra Handler later :thinking: tricky stuff